packages/assets-controllers/src/NftController.test.ts

```diff
@@ -551,12 +637,10 @@ describe('NftController', () => {
         triggerPreferencesStateChange,
         triggerSelectedAccountChange,
       } = setupController({
-        options: {
-          getERC721TokenURI: jest
-            .fn()
-            .mockImplementation(() => 'https://testtokenuri.com'),
-          getERC721OwnerOf: jest.fn().mockImplementation(() => OWNER_ADDRESS),
-        },
+        getERC721TokenURI: jest
+          .fn()
+          .mockImplementation(() => 'https://testtokenuri.com'),
+        getERC721OwnerOf: jest.fn().mockImplementation(() => OWNER_ADDRESS),
```

Comment:
These mocks are redundant with the return and resolved values being fully mocked for the `callActionSpy`. Should we rely on fully mocking the action handlers instead?